python/ray/experimental/sgd/pytorch/pytorch_distributed_sgd.py

```diff
@@ -0,0 +1,96 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import numpy as np
+import torch
+
+import ray
+
+from ray.experimental.sgd.pytorch.pytorch_runner import PyTorchRunner
+from ray.experimental.sgd.pytorch import utils
+
+
+class PyTorchDistributedSGD(object):
+    """Train a model using distributed PyTorch.
+
+    Args:
```

Comment:
These should probably be documented in the `__init__` docstring. Here, it'd make more sense to document the class attributes if relevant.

Some docstrings are missing.